### DISCUSSION TITLE
How Supademo uses GitBook

### DISCUSSION THREAD

**Discussion by aikoii** [GitbookIO/community#301] [Show and Tell]
Hey there 👋 ! Just wanted to share an article that might pique your interest. It's about how GitBook can be used for creating "how-to" articles, providing support, and onboarding new customers. You can read what Supademo shared [here](https://www.gitbook.com/blog/how-supademo-uses-gitbook-to-get-customers-up-to-speed-faster) 🌟.

**Comment by mchlkucera** [comment]
Page cannot be found @aikoii :(

**Reply by aikoii** [reply]
Hey @mchlkucera, you can check out our customer stories here: 💙 https://www.gitbook.com/customers
